### DISCUSSION TITLE
Open-Digger 是否可以去推动国内主流招聘网站（比如 BOSS 直聘）开放相关查询接口？

### DISCUSSION THREAD

**Discussion by maninhill** [X-lab2017/open-digger#778] [General] [answered]
比如：

1. 查询所有职位描述（JD）中出现某开源项目/工具的数量
2. 查询所有求职简历（CV）中含有某开源项目/工具的数量

这个数据对于评价某开源项目/工具的流行度，非常有价值。

P.S. DB-Engines 的评价标准包括这个数据，具体可以看： https://db-engines.com/en/ranking_definition

**Comment by frank-zsy** [comment, marked as answer]
That would be very useful in the future. And we may first add project dependency data and GitHub user follow data which is also very useful to indicate the importance of users and repos on network.

And if job hunting site have open API to collect the data, it will be very easy for us to collect the data and integrate into our dataset.

**Comment by frank-zsy** [comment]
We also looked into the DB-Engines ranking method and we are not very clear how they get data from LinkedIn since by our side, we found it very hard to get data from this site.

**Comment by maninhill** [comment]
Open-Digger 作为公益组织，如果能够积极努力跟 BOSS直聘等联系，定期获取到相关数据并以此制定出公开、公平的榜单，我觉得这是一件很有社会价值的事情。

**Comment by frank-zsy** [comment]
Yes, it is, we will try to reach out the job hunting sites for their data~Thank you again for your advise.
